source/developers/references/developer_guide/analytics.rst

```diff
@@ -316,24 +242,15 @@ comments that identify the purpose of the event and its fields. Your
 descriptions and examples can help assure that researchers and other members
 of the open edX community understand your intent and use the events correctly.
 
-You might find the following references helpful as you prepare your PR.
-
-* The *edX Platform Developer's Guide* provides guidelines for `contributing
-  to open edX <http://edx.readthedocs.io/projects/edx-developer-
-  guide/en/latest/process/index.html>`_.
-
-* The `edX Research
-  Guide <http://edx.readthedocs.io/projects/devdata/en/latest/>`_ is a
-  reference for information about emitted events that are included in the edX
-  tracking logs.
+You must also update the `Event Reference documentation 
+<https://docs.openedx.org/en/latest/developers/references/internal_data_formats/index.html>`_ 
+to include your changes. These documents are highly valuable for instructors, 
+researchers, site operators, and others who use the tracking logs.
 
-Request Context Middleware
-**********************************
+You might find the following references helpful as you prepare your PR.
 
-The platform includes a middleware class that enriches all events emitted
-during the processing of a given request with details about the request that
-greatly simplify downstream processing. This is called the ``TrackMiddleware``
-and can be found in ``edx-platform/common/djangoapps/track/middleware.py``.
+* The *Open edX Developer Documents* provide guidance for `contributing
+  to Open edX <https://docs.openedx.org/en/latest/developers/index.html>`_.
```

Comment:
I think if you're specifically trying to help people prepare their PR, this would be a more helpful link: https://docs.openedx.org/en/latest/developers/references/developer_guide/process/index.html

Also reminder to use Open edX as an adjective :\